source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -313,6 +319,24 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
                         telemetryProps["multilineLanguages"] = @clusterMultilineLanguages
                       end
                     end
+                    if (!@clusterNetworkFlowLogsEnabled.nil? && !@clusterNetworkFlowLogsEnabled.empty?)
+                      telemetryProps["networkFlowLogsEnabled"] = @clusterNetworkFlowLogsEnabled
+                    end
+                    if (!@clusterNetworkFlowLogsThrottleEnabled.nil? && !@clusterNetworkFlowLogsThrottleEnabled.empty?)
+                      telemetryProps["networkFlowLogsThrottleEnabled"] = @clusterNetworkFlowLogsThrottleEnabled
+                    end
+                    if (!@clusterNetworkFlowLogsThrottleRate.nil? && !@clusterNetworkFlowLogsThrottleRate.empty?)
+                      telemetryProps["networkFlowLogsThrottleRate"] = @clusterNetworkFlowLogsThrottleRate
+                    end
+                    if (!@clusterNetworkFlowLogsThrottleWindow.nil? && !@clusterNetworkFlowLogsThrottleWindow.empty?)
+                      telemetryProps["networkFlowLogsThrottleWindow"] = @clusterNetworkFlowLogsThrottleWindow
+                    end
```

Comment:
since this already covered in telemetry.go so we dont need 2 places to emit the telemetry and this can be removed.

Reply:
removed